src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int SIDE = 10;
```

Comment:
if variable is used only inside the class it's recommended to make it priave to follow the encapsulation principle

Reply:
at least got it))